bot/cogs/help_channels.py

```diff
@@ -549,6 +571,19 @@ def is_dormant_message(message: t.Optional[discord.Message]) -> bool:
                 log.trace(f"Resetting send permissions for {member} ({member.id}).")
                 await self.available_category.set_permissions(member, send_messages=None)
 
+    async def reset_send_permissions_for_help_user(self, channel: discord.TextChannel) -> None:
+        """Reset send permissions in the Available category for member that started the help session in `channel`."""
+        log.trace(f"Attempting to find user for help session in #{channel.name} ({channel.id}).")
+        try:
+            member = self.help_channel_users[channel]
+        except KeyError:
+            log.trace(f"Channel #{channel.name} ({channel.id}) not in help session cache, permissions unchanged.")
+            return
+        log.trace(f"Resetting send permissions for {member} ({member.id}).")
+        await self.available_category.set_permissions(member, send_messages=None)
```

Comment:
on an unrelated note, does that mean we can also throw them away in the Silence cog?

Reply:
Doesn't that operate on a role basis rather than on individuals? If yes, then there's only 1 role and it won't cause any harm to leave it.

Reply:
Sorry, I keep mixing up users and the main role here; came to the same realization quite a few times now... deleted the previous message before I got the response.

Reply:
Was the `send_messages` in `reset_send_permissions` intentional? This will cause the same issue after the bot loads up if I'm understanding it correctly
https://github.com/python-discord/bot/blob/7571cabe65e39d231523e713923cd23b927225bc/bot/cogs/help_channels.py#L547-L550

Reply:
An oversight. Would appreciate if you fixed that.

Reply:
Will do, this lead to my initial confusion on which was to be used